kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/writers/HazelCastWriter.scala

```diff
@@ -78,8 +71,8 @@ class HazelCastWriter(client: HazelcastInstance, settings: HazelCastSinkSettings
       logger.debug("No records received.")
     } else {
       logger.debug(s"Received ${records.size} records.")
-      val converted = records.map(r => (r.topic, convert(r)))
-      converted.foreach({
+      val grouped = records.map(r => (r.topic, r))
```

Comment:
We can do exactly the same thing as we did for Cassandra. just bare in mind order. we loose that. spawning threads even for a group we can't guarantee order.